tests/tests/group_order_api_tests.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2018 oxarbitrage, and contributors.
```

Comment:
Perhaps update this line? Does @oxarbitrage own the copyright of this file?

Reply:
80% of code is mine and I referenced oxarbitrage's code.
In such a case whose name should I put?

Reply:
changed author name to manikey123 as per discussion with @oxarbitrage